src/languageservice/utils/objects.ts

```diff
@@ -7,6 +7,15 @@
 export function equals(one: any, other: any, type?: any): boolean {
   if (type === 'boolean') {
     one = JSON.parse(one);
+    if (Array.isArray(other)) {
+      let i: number;
+      for (i = 0; i < other.length; i++) {
+        if (equals(one, other[i], type)) {
+          return true;
+        }
+      }
+      return false;
+    }
```

Comment:
Adding this array handling for a boolean is not the right thing to add here.
Iterating over the enum values should be done at the caller site (see my other comment).